emcc.py

```diff
@@ -522,27 +522,30 @@ def cxx_to_c_compiler(cxx):
   return os.path.join(dirname, basename)
 
 
-def get_binaryen_passes():
+def run_binaryen_optimizer():
```

Comment:
Maybe `should_run_binaryen_optimizer()` to make it clear that this is a predicate as opposed to the function that actually runs the optimizer.